automatic/tipp10/tipp10.nuspec

```diff
@@ -1,28 +1,42 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
-<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
-    <id>{{PackageName}}</id>
-    <version>{{PackageVersion}}</version>
-    <title>TIPP10</title>
+    <id>tipp10</id>
+    <version>2.1.0.20130528</version>
+    <packageSourceUrl>https://github.com/Chocolatey/chocolatey-coreteampackages/tree/master/manual/tipp10</packageSourceUrl>
```

Comment:
yes, I definitely will.
I just need AU to update it first since I changed it to an embedded package.